ui.go

```diff
@@ -1118,22 +1141,62 @@ func (ui *ui) readNormalEvent(ev tcell.Event) expr {
 		case tcell.ButtonNone:
 			return nil
 		}
+		if expr, ok := gOpts.keys[button]; ok {
+			return expr
+		}
 
-		expr, ok := gOpts.keys[button]
-		if !ok {
-			ui.echoerrf("unknown mapping: %s", button)
-			return draw
+		if tev.Buttons() != tcell.Button1 && tev.Buttons() != tcell.Button2 {
+			return nil
 		}
 
-		return expr
+		x, y := tev.Position()
+		wind, w := ui.winAt(x, y)
+		if wind == -1 {
+			return nil
+		}
+
+		var dir *dir
+		if gOpts.preview && wind == len(ui.wins)-1 {
+			curr, err := nav.currFile()
+			if err != nil {
+				return nil
+			} else if !curr.IsDir() {
+				return &callExpr{"open", nil, 1}
+			}
+			dir = ui.dirPrev
+		} else {
+			dir = ui.dirOfWin(nav, wind)
+			if dir == nil {
+				return nil
+			}
+		}
+
+		var file *file
+		ind := dir.ind - dir.pos + y - w.y
```

Comment:
I saw `max(dir.ind-dir.pos, 0)` elsewhere, but I'm pretty sure `dir.pos` will never be greater than `dir.ind`.